BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java

```diff
@@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
       return;
     }
 
+  // Hook to write Notifications to a temporary file on the device/emulator
+    // Keeps track of incoming notifications and makes sure count is right
+    count++ ;
```

Comment:
100 lines should be enough from testing purpose .

*"**Have you considered re-creating the file from scratch every run? is
that too much?"* :- do you mean deleting log file when app is closed ?

Currently , after every test run , i delete the log file  and create a new
one from scratch .
But i do the delete operation  from the test script itself and not in the
apk .

Still need to have  code to delete the log file after it exceeds certain
number of lines (100 ) in the apk itself .

These two above checks should be sufficient enough in my opinion .

On Fri, Jan 8, 2016 at 11:52 AM, lental <notifications@github.com> wrote:

> In
> BasicPlaybackSampleApp/app/src/main/java/com/ooyala/sample/players/BasicPlaybackVideoPlayerActivity.java
> <https://github.com/ooyala/android-sample-apps/pull/43#discussion_r49230313>
> :
>
> > @@ -95,6 +102,48 @@ public void update(Observable arg0, Object arg1) {
> >        return;
> >      }
> >
> > +  // Hook to write Notifications to a temporary file on the device/emulator
> > +    // Keeps track of incoming notifications and makes sure count is right
> > +    count++ ;
>
> if 100 lines is enough for your debugging, then that's fine with me. Have
> you considered re-creating the file from scratch every run? is that too
> much?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/android-sample-apps/pull/43/files#r49230313>.
>



-- 
Regards,

BinduMadhav Sondur
310, Crescent Village Circle , #2110 ,
San Jose , CA - 95134

PH: 315-247-5338